src/main/java/com/partymusicapp/models/PartyTemplates.java

```diff
@@ -0,0 +1,33 @@
+package com.partymusicapp.models;
+
+
+import jakarta.persistence.*;
+import lombok.Data;
+import lombok.NonNull;
+
+@Entity
+@Table(name = "PartyTemplates")
+@Data
+public class PartyTemplates {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private String tid;
```

Comment:
Eu  le-am pus nume "id" la toate, chiar daca in schema apare "pid" sau "uid" sau altele. Ii mai usor cand scrii in repos metodele care formeaza query-urile